test/known_issues/test-inspector-instrumentation-breakpoint.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+const common = require('../common');
+
```

Comment:
It would be great to add a short comment explaining what the test is testing for, and a comment that looks like this:

```javascript
// Refs: https://github.com/nodejs/node/issues/31138
```